apps/web/src/pages/Home/HomePage/components/Featured/Featured.tsx

```diff
@@ -41,32 +44,59 @@ const Featured = () => {
     ),
   };
 
+  // TEMP: Extracted for now
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const digitalAmbition = {
+    key: "digital-ambition",
+    title: intl.formatMessage({
+      defaultMessage: "The Digital Ambition",
+      id: "tTuBmE",
+      description: "Title for the digital ambition featured item",
+    }),
+    summary: intl.formatMessage({
+      defaultMessage:
+        'The Digital Ambition outlines the Government of Canada\'s commitment to create modern, accessible digital services. Achieving these priorities will result in a government that provides improved "digital-first," user-centred, and barrier-free services and programs.',
+      id: "CbzWqJ",
+      description: "Summary of the digital ambition featured item",
+    }),
+
+    img: { path: digitalAmbitionImg },
+    link: {
+      path:
+        locale === "en"
+          ? "https://www.canada.ca/en/government/system/digital-government/government-canada-digital-operations-strategic-plans/canada-digital-ambition.html"
+          : "https://www.canada.ca/fr/gouvernement/systeme/gouvernement-numerique/plans-strategiques-operations-numeriques-gouvernement-canada/ambition-numerique-canada.html",
+      label: intl.formatMessage({
+        defaultMessage: "Read the Digital Ambition",
+        id: "Gil1Zj",
+        description: "Link text to read the Digital Ambition",
+      }),
+    },
+  };
+
   // This array is just a temporary data object representing the content required by the feature blocks. This data will need to be migrated to wherever makes sense, and we'll also need dynamic routes and translated strings
```

Comment:
Does this comment still make sense?

Reply:
Removed a lot of the comments in this file in [remove extraneous comments](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6143/commits/152aa95e90c0134d6678165454934be14ae8970b)